library/src/main/java/com/anjlab/android/iab/v3/SkuDetails.java

```diff
@@ -83,72 +77,65 @@ public SkuDetails(JSONObject source) throws JSONException
         priceLong = source.optLong(Constants.RESPONSE_PRICE_MICROS);
         priceValue = priceLong / 1000000d;
         priceText = source.optString(Constants.RESPONSE_PRICE);
-        subscriptionPeriod = parsePeriod(source.optString(Constants.RESPONSE_SUBSCRIPTION_PERIOD, "P"));
-        subscriptionFreeTrialPeriod = parsePeriod(source.optString(Constants.RESPONSE_FREE_TRIAL_PERIOD, "P"));
-        haveTrialPeriod = subscriptionFreeTrialPeriod.toStandardSeconds().getSeconds() != 0;
+        subscriptionPeriod = source.optString(Constants.RESPONSE_SUBSCRIPTION_PERIOD, "P");
```

Comment:
I am happy to use the default as I did in my PR https://github.com/anjlab/android-inapp-billing-v3/pull/282/files#diff-82bb12704ee8c87bd183b059a6972bffR77, what he did was using the fallback "P" instead of the default empty "", and I left it because I thought it was more convenient to have this default for who needs this param. I will just use the default ref. https://github.com/stleary/JSON-java/blob/master/JSONObject.java#L1372